packages/docs/getting-started/running-a-validator-in-baklava.md

```diff
@@ -562,22 +567,203 @@ You can check the status of your Validator, including whether it is elected and
+
+Just like with the Validator and Vote signer, we'll want to authorize a separate Attestation signer. For that let's start our node on the Attestations machine (keep track of the password you use for this account):
+
+{% hint style="info" %}
+An earlier version of this document set the beneficiary addresses to `CELO_VALIDATOR_ADDRESS` and `CELO_VALIDATOR_GROUP_ADDRESS`, but the attestation service expects these to refer to the `ReleaseGold` contract, so please move your beneficiary addresses to reflect the environment variables above.
+
+`CELO_VALIDATOR_ADDRESS` should be your `ReleaseGold` contract address for the attestation service to work properly.
 {% endhint %}
 
+```bash
+# On the Attestation machine
+# Note that you have to export CELO_IMAGE, NETWORK_ID and CELO_VALIDATOR_ADDRESS on this machine
+docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE init /celo/genesis.json
+export BOOTNODE_ENODES="$(docker run --rm --entrypoint cat $CELO_IMAGE /celo/bootnodes)"
+docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE account new
+export ATTESTATION_SIGNER_ADDRESS=<YOUR-ATTESTATION-SIGNER-ADDRESS>
+```
+
+Let's generate the proof-of-possession for the attestation signer:
+
+```bash
+# On the Attestation machine
+docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE account proof-of-possession $ATTESTATION_SIGNER_ADDRESS $CELO_VALIDATOR_ADDRESS
+```
+
+With this proof, authorize the attestation signer on your local machine:
+
+```bash
+# On your local machine
+export ATTESTATION_SIGNER_SIGNATURE=<ATTESTATION-SIGNER-SIGNATURE>
+export ATTESTATION_SIGNER_ADDRESS=<YOUR-ATTESTATION-SIGNER-ADDRESS>
+celocli releasegold:authorize --contract $CELO_VALIDATOR_ADDRESS --role attestation --signature $ATTESTATION_SIGNER_SIGNATURE --signer $ATTESTATION_SIGNER_ADDRESS
+```
+
+You can now run the node for the attestation service in the background. In the below command remember to specify the password you used during the creation of the `ATTESTATION_SIGNER_ADDRESS` account:
+
+```bash
+# On the Attestation machine
+echo <ATTESTATION-SIGNER-PASSWORD> > .password
+docker run --name celo-attestations -it --restart always -p 127.0.0.1:8545:8545 -v $PWD:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --rpc --rpcaddr 0.0.0.0 --rpcapi eth,net,web3,debug,admin --unlock $ATTESTATION_SIGNER_ADDRESS --password /root/.celo/.password --bootnodes $BOOTNODE_ENODES
+```
+
+Next we will set up the Attestation Service itself. First, specify the following environment variables:
+
+```bash
+# On the Attestation machine
+export CELO_IMAGE_ATTESTATION=us.gcr.io/celo-testnet/celo-monorepo:attestation-service-baklava
```

Comment:
Dumb question, but this image exists, right?

Reply:
Not a dumb question, I raised this question in the description above and haven't gotten an answer - I'll try and find out.

Reply:
This image *does* exist, but was uploaded jan 10. @nambrot does it need to be updated?

Reply:
This will need to be updated with your latest fix @lucasege

Reply:
Good point, will wait until after our meeting today to verify there are no other changes but then will do